interactions:
- request:
    body: null
    headers:
      Accept:
      - '*/*'
      Accept-Encoding:
      - gzip, deflate
      Connection:
      - keep-alive
      Content-Length:
      - '0'
      User-Agent:
      - python-requests/2.32.3
      X-Traceloop-SDK-Version:
      - 1.0.0
    method: DELETE
    uri: https://api-staging.traceloop.com/v2/datasets/non-existent-dataset-123
  response:
    body:
      string: '{"error":"Dataset not found"}'
    headers:
      CF-RAY:
      - 96d8cc0a19c8c990-IAD
      Connection:
      - keep-alive
      Content-Length:
      - '29'
      Content-Type:
      - application/json; charset=utf-8
      Date:
      - Mon, 11 Aug 2025 15:34:25 GMT
      Permissions-Policy:
      - geolocation=(self), microphone=()
      Server:
      - cloudflare
      cf-cache-status:
      - DYNAMIC
      referrer-policy:
      - strict-origin-when-cross-origin
      strict-transport-security:
      - max-age=7776000; includeSubDomains
      via:
      - kong/3.7.1
      x-content-type:
      - nosniff
      x-kong-proxy-latency:
      - '0'
      x-kong-request-id:
      - 82907c482713a80d1942f12e615a603a
      x-kong-upstream-latency:
      - '4'
    status:
      code: 404
      message: Not Found
version: 1
